test/parallel/test-inspector-worker-target.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const common = require('../common');
+const fixtures = require('../common/fixtures');
+
+common.skipIfInspectorDisabled();
+
+const { NodeInstance } = require('../common/inspector-helper.js');
+
+async function setupInspector(session, sessionId = undefined) {
+  await session.send({ method: 'NodeRuntime.enable', sessionId });
+  await session.waitForNotification('NodeRuntime.waitingForDebugger');
+  await session.send({ method: 'Runtime.enable', sessionId });
+  await session.send({ method: 'Debugger.enable', sessionId });
+  await session.send({ method: 'Runtime.runIfWaitingForDebugger', sessionId });
+  await session.send({ method: 'NodeRuntime.disable', sessionId });
+  await session.waitForNotification((notification) => {
+    return notification.method === 'Debugger.scriptParsed' &&
+            notification.params.url === 'node:internal/bootstrap/realm' &&
+            notification.sessionId === sessionId;
+  });
+}
+
+async function test(isSetAutoAttachBeforeExecution) {
+  const child = new NodeInstance(['--inspect-brk=0', '--experimental-worker-inspection'],
+                                 '',
+                                 fixtures.path('inspect-worker/index.js')
+  );
+
+
+  const session = await child.connectInspectorSession();
```

Comment:
Can you add another suite to guarantee this command will fail when `--permission` is used?

Reply:
I added it below.